cpe/cpeset.py

```diff
@@ -60,14 +60,20 @@ def __getitem__(self, i):
 
         return self.K[i]
 
-    def __init__(self):
+    def __init__(self, members=[]):
```

Comment:
Using a mutable list as a default arg creates a global ... with all the issues that come with this. Are you sure that's what you wanted to do here?
instead I would use `members=None`
and below at the start of __init__ I would add:
members = members or []